triscord/settings.py

```diff
@@ -10,12 +10,12 @@ class TriscordConfigParser(configparser.ConfigParser):  # pylint: disable=R0901
 
     _loaded = False
 
-    def get(self, *args, **kwargs):
+    def get(self, *args, **kwargs):  # pylint: disable=arguments-differ
```

Comment:
Required because of ongoing bug on pylint: https://github.com/PyCQA/pylint/issues/1553